moj-node/bin/download-feedback

```diff
@@ -19,14 +19,29 @@ const mkdirp = require('mkdirp');
 
 const { getEnv } = require('../utils');
 
-const token = getEnv('MATOMO_TOKEN');
-const siteUrl = getEnv('MATOMO_URL');
+if (process.argv.length < 4) {
+  console.log(
+    '*****Please ensure you enter the command in the below format:*****',
+  );
+  console.log(
+    './bin/download-feedback [Start date YYYYMMDD],[End date YYYYMMDD]  [Establishment Name lowercase]',
```

Comment:
The dates should say `YYYY-MM-DD`

Reply:
can you change `[Start date YYYYMMDD],[End date YYYYMMDD]` to `[Date range YYYY-MM-DD,YYYY-MM-DD]` ?

Reply:
Yes agreed i can see how that may result in confusion